### DISCUSSION TITLE
About Stitching SAR frames together and Merge

### DISCUSSION THREAD

**Discussion by Obaida777** [gmtsar/gmtsar#470] [Q&A]
there are any Videos , Tutorials or clear steps with pictures beside (Time Series frome S1 TOPS:Kilauea) Tutorial and the Same with Merge if possible.

**Comment by Xiaohua-Eric-Xu** [comment]
Are you looking for an example other than this one? http://topex.ucsd.edu/gmtsar/tar/sentinel_time_series_5.pdf

**Reply by Obaida777** [reply]
Yes,please if possible

**Reply by Xiaohua-Eric-Xu** [reply]
Sorry we don't have another example for the moment. Our current short course only recorded the lecture part, but in fact, it is the breakout group meetings that deals with these processing related problems. And those are not recorded and not every student would like to work on large-scale time-series mainly due to limited access to computing resources. Would be good to follow the document but target your own area of interest. Send us questions in this post and hopefully we can get it through.
